lib/chainsync/chainsync/dashboard/usernames.py

```diff
@@ -0,0 +1,116 @@
+"""Helper functions for mapping addresses to usernames."""
+import pandas as pd
+
+
+def combine_usernames(username: pd.Series) -> pd.DataFrame:
+    """Map usernames to a single user (e.g., combine click with bots)."""
+    # TODO Hard coded mapping, should be a config file somewhere
+    user_mapping = {
+        "Charles St. Louis (click)": "Charles St. Louis",
+        "Alim Khamisa (click)": "Alim Khamisa",
+        "Danny Delott (click)": "Danny Delott",
+        "Gregory Lisa (click)": "Gregory Lisa",
+        "Jonny Rhea (click)": "Jonny Rhea",
+        "Matt Brown (click)": "Matt Brown",
+        "Giovanni Effio (click)": "Giovanni Effio",
+        "Mihai Cosma (click)": "Mihai Cosma",
+        "Ryan Goree (click)": "Ryan Goree",
+        "Alex Towle (click)": "Alex Towle",
+        "Adelina Ruffolo (click)": "Adelina Ruffolo",
+        "Jacob Arruda (click)": "Jacob Arruda",
+        "Dylan Paiton (click)": "Dylan Paiton",
+        "Sheng Lundquist (click)": "Sheng Lundquist",
+        "ControlC Schmidt (click)": "ControlC Schmidt",
+        "George Towle (click)": "George Towle",
+        "Jack Burrus (click)": "Jack Burrus",
+        "Jordan J (click)": "Jordan J",
+        # Bot accounts
+        "slundquist (bots)": "Sheng Lundquist",
+    }
+    user_mapping = pd.DataFrame.from_dict(user_mapping, orient="index")
+    user_mapping.columns = ["user"]
+    # Use merge in case mapping doesn't exist
+    username_column = username.name
+    user = username.to_frame().merge(user_mapping, how="left", left_on=username_column, right_index=True)
+    return user
+
+
+def get_click_addresses() -> pd.DataFrame:
+    """Return a dataframe of hard coded click addresses."""
+    # TODO Hard coded mapping, should be a config file somewhere
+    addresses = {
+        "0x004dfC2dBA6573fa4dFb1E86e3723e1070C0CfdE": "Charles St. Louis (click)",
```

Comment:
Same comment as above: should we move this to a non-tracked config file and programmatically generate the addresses dict? You could save the config as a json and trivially load it in.

Reply:
I'm fine with this coming in a follow-up PR